From 68d7a5aee9e88d7453b6b025573795c9a844f6c9 Mon Sep 17 00:00:00 2001
From: frq09432 <nicolas.louboutin@st.com>
Date: Mon, 29 Apr 2019 09:54:04 +0200
Subject: [PATCH] Enable DRM driver option

---
 config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config b/config
index f4992eb..ec1c706 100755
--- a/config
+++ b/config
@@ -58,7 +58,7 @@ SDK_DIR                           ?= $(AQROOT)/build/sdk
 VIVANTE_ENABLE_3D                 ?= 1
 VIVANTE_ENABLE_2D                 ?= 1
 VIVANTE_ENABLE_VG                 ?= 1
-VIVANTE_ENABLE_DRM                ?= 0
+VIVANTE_ENABLE_DRM                ?= 1
 NO_DMA_COHERENT                   ?= 0
 USE_PLATFORM_DRIVER               ?= 1
 ENABLE_GPU_CLOCK_BY_DRIVER        ?= 1
-- 
2.7.4

